calendar-app/Components/GroupSection/InGroupSection.jsx

```diff
@@ -6,14 +6,21 @@ import { loginUserAPI } from '../API/Users/UsernameCheck';
 import { inviteToGroup } from '../API/Invites/InviteToGroup';
 import { getGroupMembers } from '../API/Groups/GroupMembers';
 import { getGroupId } from "../Storage/userDataStorage";
+import { getUserId } from "../Storage/userDataStorage";
```

Comment:
this can be wrote in line 8 like import { getGroupId, getUserId } from "../Storage/userDataStorage";